release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -20,6 +20,8 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 
 {product-title} {product-version} is based on {op-system-base-full} 9.2. {op-system-base} 9.2 has not yet been submitted for FIPS validation. Red Hat expects, though cannot commit to a specific timeframe, to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Updates will be available in link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards].
 
+Because {product-title} 4.13 is based on the RHEL 9.2 host operating system, the micro-architecture requirements are now increased to x86-64-v2, Power9, and Z14. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#architectures[RHEL micro-architecture requirements documentation]. You can verify compatibility before updating by following the procedures outlined in this link:https://access.redhat.com/solutions/7052996[KCS article].
```

Comment:
```suggestion
Because {product-title} 4.13 is based on the {op-system-base} 9.2 host operating system, the micro-architecture requirements are now increased to x86_64-v2, Power9, and Z14. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#architectures[RHEL micro-architecture requirements documentation]. You can verify compatibility before updating by following the procedures outlined in this link:https://access.redhat.com/solutions/7052996[KCS article].
```
https://redhat-documentation.github.io/supplementary-style-guide/#x86_64 for the arches

Reply:
Since this is an addition to the main body of the release notes, does this need change management?

Reply:
Nope! I added to the additional notes section of the PR - I checked with @kalexand-rh , @sferich888 , @sjstout and @sdodson and they all agreed that we do not need change management here.